exercises/4-an-enslaved-hope/README.md

```diff
@@ -67,19 +68,18 @@ _____
 ```
 
 3. Create the object for feeding the template with the parameters
- 
+
+<kbd>*inventory/host_vars/ci-cd-tooling.yml*</kbd>
 ```yaml
-# inventory/host_vars/ci-cd-tooling.yml
-
-  - object: jenkins-slave-nodes
-    content:
-      - name: jenkins-slave-zap
-        template: "{{ cop_quickstarts_raw }}/{{ cop_quickstarts_raw_version_tag }}/jenkins-slaves/.openshift/templates/jenkins-slave-generic-template.yml"
-        params_from_vars: "{{ zap }}"
-        namespace: "{{ ci_cd_namespace }}"
-        tags:
-        - jenkins-slaves
-        - zap-slave
```

Comment:
Need to be careful about the indentation changes here ... the copy/paste will be wrong.